src/browser/components/MainPage.jsx

```diff
@@ -79,14 +79,25 @@ export default class MainPage extends React.Component {
   }
 
   getTabWebContents(index = this.state.key || 0, teams = this.props.teams) {
+    const allWebContents = remote.webContents.getAllWebContents();
+    const openDevTools = allWebContents.find((webContents) => webContents.getURL().includes('chrome-devtools') && webContents.isFocused());
+    if (openDevTools) {
+      return openDevTools;
+    }
+
+    if (this.state.showNewTeamModal) {
+      const indexURL = remote.getGlobal('isDev') ? 'http://localhost:8080/browser/index.html' : `file://${remote.app.getAppPath()}/browser/index.html`;
+      return allWebContents.find((webContents) => webContents.getURL().includes(indexURL));
```

Comment:
Guys, what if we only use [getfocusedwebcontents](https://electronjs.org/docs/api/web-contents#webcontentsgetfocusedwebcontents) here?

There was an old bug with electron that may be the reason to avoid it but now I can see cut, copy and paste working as expected in debug.

@devinbinnie 
In which case it picks up the web view? I may be missing something here.

Reply:
Just to be clear, I'm talking about **not** using `indexURL` at all, and just returning the current focused webcontents whether it's dev tools, mainpage or any modal.

Reply:
@xalkan `getFocusedWebContents` doesn't work for the MainPage at all. It always grabs the last webview (ie. your last tab). I think it's better to be explicit as we have been.